test/test_linalg.py

```diff
@@ -8082,6 +8082,26 @@ def test_tensordot(self, device):
         an = torch.from_numpy(np.tensordot(np.zeros((), dtype=np.float32), np.zeros((), dtype=np.float32), 0))
         self.assertEqual(a, an)
 
+    @onlyCUDA
+    @skipCUDAIfRocm
```

Comment:
Is this test failing for ROCm? Sure "cuSOLVER" backend is not available there, but in this case, it should probably pick the "default" backend, no?

Reply:
Actually, yes, you're right. 😅  It'll pick the default magma path. Let me remove this rocm skip.